README.md

```diff
@@ -141,6 +141,11 @@ named arguments:
   -s {true,false}, --skipprcreation {true,false}
                          Only update image tag store. Skip creating PRs
   -x X                   comment snippet mentioned in line just before FROM instruction for ignoring a child image. Defaults to 'no-dfiu'
+  -u, --useratelimiting  {true,false}. Defaults to false, means no ratelimiting is imposed if this is not set to true
+                         Use RateLimiting for throttling the number of PRs DFIU will cut over a period of time. Throttling works based on token-bucket algorithm.
```

Comment:
Users shouldn't be exposed to what algo is used for throttling. We can put in this in md docs in the repo, but not in the cmd-line interface